specification/dimensions/listunitprice.md

```diff
@@ -0,0 +1,32 @@
+# List Unit Price
+
+The List Unit Price represents the suggested provider published price for a single pricing measurement unit, which incorporates volume/tier-based reductions while excluding any negotiated or commitment-based discounts. This price is denominated in the [Billing Currency](#billingcurrency). It is used to calculate List Cost and savings based on rate optimization activities.
+
+The BilledCost column MUST be present in the billing data. This column MUST be a numeric value of type Decimal and MUST NOT contain null values.
```

Comment:
Looks like BilledCost related sentence got in some how. I assume this was put so we can make the line below consistent with the other 'Metric' columns.

Reply:
Introduced by mistake :(. Deleted it.